announcements/2021/week14.md

```diff
@@ -0,0 +1,16 @@
+# :zap: This week in provider-independent announcements (April 9, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+[Cert Manager v2.4.4](https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.4.4/) renames the clusterissuer subchart and makes pods of deployments use read-only file systems, in order to...
+
+[External DNS v.2.3.0](https://docs.giantswarm.io/changes/managed-apps/external-dns-app/v2.3.0/) now periodically validates IAM credentials for AWS, self-killing and recovering from authentication errors.
```

Comment:
@ubergesundheit can you confirm if this is correct? 😄

Reply:
Yes.

Can you remind me who is the main target of these announcements? If its for customers, this is not really interesting

Reply:
since you confirmed, this wont affect customers and not interesting to them, deleted it!